src/locale/index.js

```diff
@@ -13,7 +13,10 @@ import en_US from './en_US.js';
 
 export const LocaleContext: React.Context<LocaleT> = React.createContext(en_US);
 
-const LocaleProvider = (props: {locale: LocaleT, children: ?React.Node}) => {
+const LocaleProvider = (props: {
+  locale: $Shape<LocaleT>,
```

Comment:
shape marks all fields as optional, which is not correct. type spread will coerce an exact type to inexact `{...LocaleT}`

Reply:
This is intentional. We are doing a deep merge with the defaults (English). You should be able to pass only a part of locale file (like a single component). This was breaking the doc example.

Reply:
ah right 👍